### ISSUE TITLE
Separate Tracking of DEMO, TEAM & PROD SIM UI Web Pages in Google Analytics

### ISSUE THREAD

**Issue by jamesmrollins** [lzim/teampsd#1052]
**1. Add description:** 
Can Google Analytics be set up to track the MTL Demo and SIM UI webpages seperately?

<< Paste screenshots here>>
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by lzim** [comment]
Thanks @jamesmrollins 

Sorry about any miscommunication, but real ask here is a Google Analytics account for all 3 webpages:
1. PROD - set up
2. DEMO - set up
3.  TEAM - not set up.

Thanks!

**Comment by jamesmrollins** [comment]
@lzim  WILCO!

**Comment by lzim** [comment]
Thanks 👍🏽

**Comment by jamesmrollins** [comment]
@hirenp-waferwire let's get online together and set up the https://forio.com/app/va/va-psd-team/teampsd.html website for tracking on Google Analytics.
